department-of-veterans-affairs/abd-vro
# Mcp 2334 - Remove BIP Controller

Remove BIP Controller

## What was the problem?
The BIP Controller was put there for testing BIP API service access as we have no BIP API connection in our development environment. As we passed the local environment testing process for BIP API access, we can remove BIP controller.

Associated tickets or Slack threads:
MCP-2334

## How does this fix it?[^1]
Remove the code.

## How to test this PR
- Step 1. Build the app
- Step 2. BIP Controller should be shown in swagger ui.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
